sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -15,27 +15,29 @@ tag: package-2021-02
 output-folder: $(this-folder)/Generated
 clear-output-folder: true
 skip-csproj: true
+mgmt-debug:
+  show-serialized-names: true
 modelerfour:
   flatten-payloads: false
   lenient-model-deduplication: true
```

Comment:
can remove this config by rename dup models

Reply:
remove this config will cause the following error: his error can be *temporarily* avoided by using the 'modelerfour.lenient-model-deduplication' setting.  NOTE: This setting will be removed in a future version of @autorest/modelerfour; schemas should be updated to fix this issue sooner than that.